src/client/Microsoft.Identity.Client/Extensibility/AbstractAcquireTokenParameterBuilderExtension.cs

```diff
@@ -30,5 +30,28 @@ public static partial class AbstractConfidentialClientAcquireTokenParameterBuild
 
             return builder;
         }
+
+        /// <summary>
+        /// Binds the token to a key in the cache. L2 cache keys contain the key id.
```

Comment:
Please add a comment that this extensibility point is not meant to be used except by other Identity SDKs.